test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -70,6 +70,17 @@ def create_mom_resources(self, resc_name, resc_type, resc_flag,
             self.filenames.append(fp)
         return fp_list
 
+    def check_access_log(self, fp, exist=True):
+        match_from = time.time()
```

Comment:
if you decide to int() the value in server_dyn_res test, then please int() this value as well